### ISSUE TITLE
Hard power cap?

### ISSUE THREAD

**Issue by rollthedaisuke** [Ethanol10/Ror2Ganon#2]
I can't seem to pick up items in the second stage. I've only played two games with the king of evil, and both times the second stage has been the abandoned aqueduct so maybe it's specific to that stage, or just outside of the first stage in general. Ganon do be stronk, so I can see this being a feature, however it would be nice to become a god king of evil.

**Comment by rollthedaisuke** [comment]
I have now realized that I can actually infinitely gain the benefits of items, without actually consuming them. Spammed interact on a goat hoof, tapped W, and teleported across the map. Pretty nice.

**Comment by Ethanol10** [comment]
Sorry, what's the issue?

**Comment by rollthedaisuke** [comment]
When playing as ganon, either on the second stage of a run or specifically within the abandoned aqueduct, I cannot pick up items, however I still gain their benefits (I think) every frame that the model is in contact with the item. See the attached run where I somehow ended up with 728 Paul's Goat Hoofs and over a thousand armor plates. I don't think the plates actually took effect, since I let myself be killed by basic mobs once zooming around the map got old.
![OP](https://user-images.githubusercontent.com/113219325/189464557-be7e9808-bbbb-4f6c-a477-1f6a0d91a467.png)

**Comment by rollthedaisuke** [comment]
I would love to help you resolve this, as I love your implementation of Ganondorf into the game. I never knew I wanted to stomp wisps until I was granted that ability.

**Comment by Ethanol10** [comment]
Huh, that's a strange issue, just in case what's your mod list? Do you have logs too? From the time you played that run.

**Comment by Ethanol10** [comment]
@rollthedaisuke Sorry, on trying to use a default mod list, I can't seem to replicate the issue.

**Comment by rollthedaisuke** [comment]
[rollthedaisuke ganon log.zip](https://github.com/Ethanol10/Ror2Ganon/files/9539948/rollthedaisuke.ganon.log.zip)
![modlist](https://user-images.githubusercontent.com/113219325/189477661-1d1fb0d7-a0e4-4114-97e9-71622b8b986e.png)

I came back to the game after a good bit of time away from it and saw "holydll" which claimed to improve performance so I blindly installed it. As it's the only new mod I'm using beside the character mod, I tried disabling it and the issue is resolved. Literally one of the first things it says within the description is "THIS MOD CAN BREAK COMPATIBILITY WITH OTHER MODS,DONT COMPLAIN TO OTHER MOD AUTHORS IF SOMETHING BREAKS WITH THIS." So clearly I should have checked that first.
